src/vcModals.cpp

```diff
@@ -325,24 +338,42 @@ void vcModals_DrawTileServer(vcState *pProgramState)
     static bool s_isDirty = false;
     static int s_currentItem = -1;
 
-    const char *pItems[] = { "png", "jpg" };
+    const char *pSlippyItems[] = { "png", "jpg" };
+    const char *pGoogleItems[] = { "Plan","Image","Terrain" };
     if (s_currentItem == -1)
     {
-      for (int i = 0; i < (int)udLengthOf(pItems); ++i)
-      {
-        if (udStrEquali(pItems[i], pProgramState->settings.maptiles.tileServerExtension))
-          s_currentItem = i;
-      }
+        if (pProgramState->settings.maptiles.serverFlag == vcMTSF_Slippy)
+        {
+            for (int i = 0; i < (int)udLengthOf(pSlippyItems); ++i)
+            {
+              if (udStrEquali(pSlippyItems[i], pProgramState->settings.maptiles.tileServerExtension))
+              {
+                s_currentItem = i;
+                break;
+              }
+            }
+        }
+        else if (pProgramState->settings.maptiles.serverFlag == vcMTSF_Google)
+        {
+            s_currentItem = pProgramState->settings.maptiles.mapTileType;
+        }
     }
 
     if (ImGui::InputText(vcString::Get("settingsMapsTileServer"), pProgramState->settings.maptiles.tileServerAddress, vcMaxPathLength))
       s_isDirty = true;
 
-    if (ImGui::Combo(vcString::Get("settingsMapsTileServerImageFormat"), &s_currentItem, pItems, (int)udLengthOf(pItems)))
+    if (pProgramState->settings.maptiles.serverFlag == vcMTSF_Slippy &&
+        ImGui::Combo(vcString::Get("settingsMapsTileServerImageFormat"), &s_currentItem, pSlippyItems, (int)udLengthOf(pSlippyItems)))
```

Comment:
We don't break an `if` conditional across multiple lines unless absolutely neccessary - which it isn't in this case. Please put it all back on one line.